benchmark/benchmark_basic_english_normalize.py

```diff
@@ -0,0 +1,36 @@
+import time
+
+import torch
+from torchtext.experimental.datasets import AG_NEWS
+from torchtext.experimental.transforms import BasicEnglishNormalize
+from torchtext.data.utils import get_tokenizer
+
+
+def benchmark_experimental_vocab():
+    def _run_benchmark_lookup(train, tokenizer):
+        t0 = time.monotonic()
+        for (label, text) in train.data:
+            tokenizer(text)
+        print("Lookup time:", time.monotonic() - t0)
+
+    train, = AG_NEWS(data_select='train')
```

Comment:
Since you are calling `torchtext.experimental.datasets.AG_NEWS`, do you receive the processed data (a.k.a. tensor)? Should we use `torchtext.experimental.datasets.raw.AG_NEWS`?

Reply:
I was doing `train.data` before to get the `(label, text)` tuple but I guess using an iterator for the raw dataset may be the better way to go about it.